recipes/longestrunsubsequence/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.0.1" %}
+
+package:
+  name: "longestrunsubsequence"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/AlBi-HHU/longest-run-subsequence/archive/v{{ version }}.tar.gz
+  sha256: 75440b9097ee018762febffd2495be3bd24ccbf3642643e1ed7e9b1cb157d76c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install --no-deps -vv ."
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - python >=3.5
+  run_constrained:
```

Comment:
Please remove the `run_constrained:` entry, it isn't needed. Having `pulp >=2` is sufficient.

Reply:
Thanks! My understanding of run_constrained is that it's meant to specify optional dependencies. longestrunsubsequence uses pulp if it detects its presence, but has no hard dependency on it. What's the proper way to specify this?

Reply:
OK, that should presumably work then. We usually just add optional requirements that we expect users will want as normal run-time dependencies.